linkis-public-enhancements/linkis-publicservice/linkis-basedata-manager/src/main/java/org/apache/linkis/basedatamanager/server/service/impl/DatasourceEnvServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.basedatamanager.server.service.impl;
+
+import org.apache.linkis.basedatamanager.server.dao.DatasourceEnvMapper;
+import org.apache.linkis.basedatamanager.server.domain.DatasourceEnvEntity;
+import org.apache.linkis.basedatamanager.server.service.DatasourceEnvService;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
+import com.github.pagehelper.PageHelper;
+import com.github.pagehelper.PageInfo;
+
+/**
+ * @author jack
+ * @description 针对表【linkis_ps_dm_datasource_env】的数据库操作Service实现
```

Comment:
Apache Linkis is licensed under the Apache License Version 2.0, which
does not allow using author tag in java code. All listed author info
can be gained from git commit history. detail info can be foud https://lists.apache.org/thread/dkt05nx61phkhhnbsbmc3lvh8p10rdq8

Reply:
It is recommended to use English first in the annotation code